lib/services/android-project-service.ts

```diff
@@ -39,7 +39,7 @@ class AndroidProjectService extends projectServiceBaseLib.PlatformProjectService
 	public get platformData(): IPlatformData {
 		if (!this._platformData) {
 			let projectRoot = path.join(this.$projectData.platformsDir, "android");
-
+			let packageName = this.getProjectNameFromId(this.$projectData.projectId);
```

Comment:
You can use `this.$projectData.projectId` inside `getProjectNameFromId` function